.github/workflows/package.yml

```diff
@@ -0,0 +1,36 @@
+name: Package
+
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  Package:
+    name: Package and Publish Docker Image
+    if: github.event.base_ref == 'refs/heads/master'
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - name: Get Version
+      id: get_version
+      run: |
+        echo ::set-output name=version::${GITHUB_REF/refs\/tags\/}
```

Comment:
Do we need to escape the first forward slash too?
```
echo ::set-output name=version::${GITHUB_REF/refs\/tags\/}
                                            ^
```

Reply:
The prebid server implementation doesn't escape the first slash, so seems correct.